tools/ci_build/github/windows/post_code_coverage_to_dashboard.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+
+# command line arguments
+# --report_url=<string>
+# --report_file=<string, local file path>
+# --commit_hash=<string, full git commit hash>
+
+import argparse
+import mysql.connector
+import xml.etree.ElementTree as ET
+import sys
+import os
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description="ONNXRuntime test coverge report uploader for dashboard")
+    parser.add_argument("--report_url", help="URL to the Cobertura XML report")
+    parser.add_argument("--report_file", help="Path to the local cobertura XML report")
+    parser.add_argument("--commit_hash", help="Full Git commit hash")
+    return parser.parse_args()
+
+def parse_xml_report(report_file):
+    tree = ET.parse(report_file) # may throw exception
+    root = tree.getroot()
+    result = {}
+    
+    result['coverage'] = float(root.get('line-rate'))
+    result['lines_covered'] = int(root.get('lines-covered'))
+    result['lines_valid'] = int(root.get('lines-valid'))
+    return result
+
+def write_to_db(coverage_data, args):
+    # connect to database
+
+    cnx = mysql.connector.connect(
+        user='ort@onnxruntimedashboard', 
+        password=os.environ.get('DASHBOARD_MYSQL_ORT_PASSWORD'), 
+        host='onnxruntimedashboard.mysql.database.azure.com', 
+        database='onnxruntime')
+
+    try:
+        cursor = cnx.cursor()
+
+        #delete old records
+        delete_query = ('DELETE FROM onnxruntime.test_coverage '
+            'WHERE UploadTime < DATE_SUB(Now(), INTERVAL 30 DAY);'
+        )
+        
+        cursor.execute(delete_query)
+
+        #insert current record
+        insert_query = ('INSERT INTO onnxruntime.test_coverage '
+            '(UploadTime, CommitId, Coverage, LinesCovered, TotalLines, ReportURL) '
+            'VALUES (Now(), "%s", %f, %d, %d, "%s") '
+            'ON DUPLICATE KEY UPDATE '
```

Comment:
Now it seems the database has duplicated data. On the last day, our code coverage is more than 100%.

Reply:
Some dummy info I added for debugging, those are cleaned up now. Coverage is ~ 81%